Kernel/System/DynamicField.pm

```diff
@@ -773,20 +773,20 @@ sub DynamicFieldList {
 
             if ( IsStringWithData( $Param{Namespace} ) && $Param{Namespace} ne 'All' ) {
                 push @WhereClauses, 'name LIKE ?';
-                push @BindValues,   "$Param{Namespace}-%";
+                push @Bind,         \"$Param{Namespace}-%";
             }
 
         }
 
-        if ( @WhereClauses && @BindValues ) {
+        if ( @WhereClauses && @Bind ) {
```

Comment:
The check for @Bind is a bit strange as it isn't used in the if-block.

Reply:
You're right - I changed the code accordingly. Thank you :)